MAUI/Segmented-Control/customization.md

```diff
@@ -0,0 +1,395 @@
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.TextStyle>
+            <buttons:SegmentTextStyle TextColor="Red" FontAttributes="Italic" FontSize="20"/>
+        </buttons:SfSegmentedControl.TextStyle>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.TextStyle = new SegmentTextStyle()
+        {
+            TextColor = Colors.Red,
+            FontAttributes = FontAttributes.Italic,
+            FontSize = 20,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment item
+
+#### Customizing the segment corner radius
+
+You can use `SegmentCornerRadius` property of `SfSegmentedControl`, to customize the segment corner radius of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                SegmentCornerRadius="5">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SegmentCornerRadius = 5;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customizing the segment background
+
+You can use `SegmentBackground` property of `SfSegmentedControl`, to customize the segment background of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                SegmentBackground="LightGray">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SegmentBackground = Colors.LightGray;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing disabled segment item
+
+You can restrict selection and interaction of segment item by setting `IsEnabled` property of `SfSegmentItem` to `false`. The value `IsEnabled` property is `true` by default.
+
```

Comment:
in disabled segment title add this
SetSegmentEnabled

Reply:
Added SetSegmentEnabled